site/_components/card-subnav.njk

```diff
@@ -4,7 +4,9 @@
   @param {object} navSection - Object containing the following attributes.
   @param {string} navSection.title - Nav section title.
   @param {string} navSection.desc - Nav section description.
-  @param {string} navSection.icon - Nav section icon.
+  @param {object} navSection.icon (optional) - Object containing the following attributes.
+  @param {string} navSection.icon.name - ID of icon within spritesheet.
+  @param {('primary-blue'|'primary-yellow'|'primary-green')} navSection.icon.background - Color keyword to apply as the icon's background circle.
```

Comment:
For simplicity's sake, we should make this just `blue`, `yellow`, or `green`. We can interpolate what the color should be from those, and we don't expose our CSS variables to our config, letting what particular shade we use be changeable without coupling it to our invocations or CSS variables directly